packages/app/src/containers/preview-area.js

```diff
@@ -133,15 +133,22 @@ export const PreviewArea = ({ isEditor }) => {
                 width: '24px',
                 height: '24px',
                 background:
-                  "url('https://cdn.microlink.io/logo/logo.svg') no-repeat center center / 22px"
+                  "url('https://cdn.microlink.io/logo/logo.svg') no-repeat center center / 22px",
+                fontSize: '0px',
+                color: 'transparent',
+                userSelect: 'none'
               }}
-            />
+              rel=''
+            >
+              Microlink – Browser as API
+            </ExternalLink>
             {[
               {
                 href:
                   'https://microlink.io/docs/cards/getting-started/overview',
                 children: 'Docs',
-                title: 'Read documentation'
+                title: 'Read documentation',
+                rel: ''
```

Comment:
not sure why rel is empty 🤔

Reply:
Because the component has a default value for `rel`. I did this to those 2 anchors because they're "internal" links (or at least to the same domain), so you probably want to keep the referrer - and we don't have to worry about noopener.